pom.xml

```diff
@@ -314,6 +314,8 @@
         <failsafe.argline>${jacoco.argline}</failsafe.argline>
 
         <fabric8.docker.plugin.version>0.27.1</fabric8.docker.plugin.version>
+        <tink.version>1.2.2</tink.version>
+        <org.json.version>20170516</org.json.version>
```

Comment:
❓ We updated the `protobuf.version` to `3.6.1` on the release branch. Should we not also update it from `3.5.1` to `3.6.1` here on `master`?

Reply:
Great catch! I forgot `3.6.1` was the correct version 😅